fastlane.gemspec

```diff
@@ -78,7 +78,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency('plist', '>= 3.1.0', '< 4.0.0') # Needed for set_build_number_repository and get_info_plist_value actions
   spec.add_dependency('CFPropertyList', '>= 2.3', '< 4.0.0') # Needed to be able to read binary plist format
   spec.add_dependency('addressable', '>= 2.8', '< 3.0.0') # Support for URI templates
-  spec.add_dependency('multipart-post', '~> 2.0.0') # Needed for uploading builds to appetize
+  spec.add_dependency('multipart-post', '>= 2.0.0', '< 3.0.0') # Needed for uploading builds to appetize
```

Comment:
Initially, I thought `~> 2.0.0` and `>= 2.0.0, < 3.0.0` were equivalent, but I see the difference now. I suppose we could change `~> 2.0.0` to `~> 2`, which is simpler, but that would go against the code style pattern being used nearly-throughout this file so I think this is better :)

Reply:
Yes, I followed the style I saw in the rest of that `.gemspec` file.